src/components/Avatar/Avatar.css

```diff
@@ -49,16 +50,22 @@
   width: 20px;
   bottom: -2px;
   right: -2px;
-  overflow: hidden;
   border-radius: 50%;
   border: solid 2px #fff;
+  background: #fff;
+  color: $color__river;
 }
 .y-avatar__size-small--badge,
 .y-avatar__size-xSmall--badge {
   height: 16px;
   width: 16px;
 }
 
+.y-avatar--badge > .y-icon {
+  top: 0;
+  display: block;
```

Comment:
we'll wanna use Icon prop block=true here

Reply:
Does it keep the fix visually? I'm not familiar with what that does

Reply:
thats fine. I'll make the change and the visuals shouldnt change.